### ISSUE TITLE
Can't build the project when we update to vs 2019 preview 4.0 

### ISSUE THREAD

**Issue by BenLampson** [dotnet/aspnetcore#23951]
```
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.7.0 Preview 4.0

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19041
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.100-preview.6.20318.15\

Host (useful for support):
  Version: 5.0.0-preview.6.20305.6
  Commit:  4ba9ecaabd

```

```
.NET SDKs installed:
  3.1.301 [C:\Program Files\dotnet\sdk]
  3.1.400-preview-015203 [C:\Program Files\dotnet\sdk]
  5.0.100-preview.6.20318.15 [C:\Program Files\dotnet\sdk]

1>------ Build started: Project: DevelopmentCode, Configuration: Debug Any CPU ------
1>You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
1>Unknown C# language version 9.0.
1>C:\Program Files\dotnet\sdk\5.0.100-preview.6.20318.15\Sdks\Microsoft.NET.Sdk.Razor\build\netstandard2.0\Microsoft.NET.Sdk.Razor.Component.targets(106,5): error : rzc generate exited with code 2.
1>Done building project "DevelopmentCode.csproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 5 up-to-date, 0 skipped ==========
```

I fix this use the step below :

``` xml
  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
      <LangVersion>8.0</LangVersion>
  </PropertyGroup>
```

Change the lang version to 8.0

**Comment by javiercn** [comment]
@BenLampson thanks for contacting us. 

Does this happen for a new project?

**Comment by BenLampson** [comment]
Yes~ It happens for a new project.

**Comment by JoakimOgren** [comment]
I have samme issue.

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us. The workaround you've got is what you can use for now. This issue has been addressed and will be released as part of 5.0 Preview 7 SDK.
